addons/website_jitsi/static/src/js/chat_room.js

```diff
@@ -4,63 +4,24 @@ import publicWidget from "@web/legacy/js/public/public_widget";
 import { renderToElement } from "@web/core/utils/render";
 import { utils as uiUtils } from "@web/core/ui/ui_service";
 import { rpc } from "@web/core/network/rpc";
+import { patch } from "@web/core/utils/patch";
 
-publicWidget.registry.ChatRoom = publicWidget.Widget.extend({
-    selector: '.o_wjitsi_room_widget',
-    events: {
-        'click .o_wjitsi_room_link': '_onChatRoomClick',
-    },
-
-    /**
-      * Manage the chat room (Jitsi), update the participant count...
-      *
-      * The widget takes some options
-      * - 'room-name', the name of the Jitsi room
-      * - 'chat-room-id', the ID of the `chat.room` record
-      * - 'auto-open', the chat room will be automatically opened when the page is loaded
-      * - 'check-full', check if the chat room is full before joining
-      * - 'attach-to', a JQuery selector of the element on which we will add the Jitsi
-      *                iframe. If nothing is specified, it will open a modal instead.
-      * - 'default-username': the username to use in the chat room
-      * - 'jitsi-server': the domain name of the Jitsi server to use
-      */
-    start: async function () {
-        await this._super.apply(this, arguments);
-        this.roomName = this.$el.data('room-name');
-        this.chatRoomId = parseInt(this.$el.data('chat-room-id'));
-        // automatically open the current room
-        this.autoOpen = parseInt(this.$el.data('auto-open') || 0);
-        // before joining, perform a RPC call to verify that the chat room is not full
-        this.checkFull = parseInt(this.$el.data('check-full') || 0);
-        // query selector of the element on which we attach the Jitsi iframe
-        // if not defined, the widget will pop in a modal instead
-        this.attachTo = this.$el.data('attach-to') || false;
-        // default username for jitsi
-        this.defaultUsername = this.$el.data('default-username') || false;
-
-        this.jitsiServer = this.$el.data('jitsi-server') || 'meet.jit.si';
+patch(publicWidget.registry.ChatRoom.prototype, {
 
-        this.maxCapacity = parseInt(this.$el.data('max-capacity')) || Infinity;
-
-        if (this.autoOpen) {
-            await this._onChatRoomClick();
-        }
-    },
 
     //--------------------------------------------------------------------------
     // Handlers
     //--------------------------------------------------------------------------
 
     /**
-      * Click on a chat room to join it.
-      *
-      * @private
-      */
+     * @override
+     */
     _onChatRoomClick: async function () {
+        this.jitsiServer = this.roomUrl;
```

Comment:
I kept `jitsiServer` to avoid confusion as it is more a server url than a roomUrl, 
Don't you think that aving : 
`window.location = 'intent://${this.chatRoomUrl}/${encodeURIComponent(roomName)}#Intent......'`
or
`url: https://${this.chatRoomUrl}/external_api.js`
would be a bit strange ?